packages/libllvm/build.sh

```diff
@@ -81,6 +82,13 @@ termux_step_host_build() {
 }
 
 termux_step_pre_configure() {
+	# Version guard to keep flang in sync
```

Comment:
This check cannot be placed here. If it is placed here, the revision of flang needs to be increased by 1 and the revision of libllvm needs to be increased by 2, otherwise CI will fail to upload the package to aptly.

Reply:
The version should be fine, as there is no reason to upload this libllvm package again just for this build version check.

The only issue I thought of is that modifying this script used to trigger a rebuild of libllvm just because these lines were added, which might have timed out the CI, but it looks like that Termux build logic has changed because a rebuild of libllvm was not triggered.

Reply:
flang, libllvm, vim, vim-python, nfs-utils, vim-gtk were all rebuilt in this CI run, maybe github actions are faster than they used to?

Can push libllvm update in separate commit/PR with `%ci:no-build` to not have to change revisions and so on

Reply:
(I guess vim and friends were built because branch hasn't been rebased)

Reply:
@licy183 Thanks, nicely spotted :eyes: !

Have broken out the `libllvm` comment and version guard to #20606 with a `%ci:no-build`.

Reply:
> flang, libllvm, vim, vim-python, nfs-utils, vim-gtk were all rebuilt in this CI run, maybe github actions are faster than they used to?

You sure, I didn't [see it in the build log](https://github.com/termux/termux-packages/actions/runs/9581036040/job/26416933965#step:8:710)? The CI isn't _that_ fast. 😉

Reply:
I still have some doubts about this approach... Is it possible to compile libllvm and flang in the same PR-triggered action runner and not reach the 6-hour time limit?

Reply:
Ah... Seems OK. https://github.com/termux/termux-packages/pull/20611